### PR TITLE
[#9740] Fix Instructor Homepage UI

### LINKED ISSUE #9740: Instructor Homepage: "Add New Course" and "Sort By" loose responsiveness at Mobile Widths
At mobile and smaller screen widths, the "Add New Course" button and Sort By text overlap, making both unreadable and the "Add New Course" button unusable where there is overlap.

**Environment**: Live Server V6.16.2

**Steps to reproduce**
- On a phone or at browser widths smaller than 530px wide, log in as instructor
- Navigate to Home page
- Observe overlap of "Add New Course" button and "Sort By" text

**Expected behaviour**
"Add New Course" and "Sort By" buttons should stack vertically at xs screen widths

**Actual behaviour**
"Add New Course" and "Sort By" buttons overlap at xs screen widths

**Additional info**
Observed Behavior in Chrome 74

image

### PR DESCRIPTION
**Pull request by madanalogy**
Fixes #9740 

Removes static height definition for row to allow responsiveness in extra small screen sizes. Fix verified in local dev environment.

### CODE REVIEW

**Comment by xpdavid** [comment]
Hi, @madanalogy Can you post a screenshot of the new UI after your changes?

**Comment by madanalogy** [comment]
> Hi, @madanalogy Can you post a screenshot of the new UI after your changes?

Here you go :)
Screenshot 2019-12-12 at 1 42 53 AM

File: src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h1>Home</h1>
   </ng-container>
 </ng-template>
 
-<div class="row height-60px">
```
Comment (xpdavid) [inline src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html]: @madanalogy I think you can remove this CSS class in `instructor-home-page.component.scss`

Comment (madanalogy) [reply src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html]: Roger that, css declaration removed. Thanks for the suggestion!

**Pull request merged**